scapy/sendrecv.py

```diff
@@ -16,7 +16,7 @@
 import subprocess
 import time
 import types
-
+import socket
```

Comment:
Also please move it to its right alphabetical place in the imports (above: native modules). Below is reserved for scapy modules